Src/StartMenu/StartMenuDLL/LogManager.cpp

```diff
@@ -51,3 +53,31 @@ void LogMessage( const wchar_t *text, ... )
 
 	fflush(g_LogFile);
 }
+
+void LogPropertyStore(TLogCategory category, IPropertyStore* store)
+{
+	if (!store)
+		return;
+
+	DWORD count = 0;
+	store->GetCount(&count);
+	for (DWORD i = 0; i < count; i++)
```

Comment:
a for loop in a logging function? Mhmm.. okay.

Reply:
The function is about logging each property in property store. For debugging purposes.